Add updateOnlyNotNullProperties boolean in order to update only not null properties

Hi,

I'm trying to perform partial updates based on the fact that `ObjectMappingConfiguration.serializeNull = false`. So i expect that my Document doesn't serialize field2 when it's value is null. Obviously i'm using kmongo-native

Am I interpreting this feature wrong?? btw no documentation on this topic anywhere. Here is a Unit test I've made explaining the behavior

```kotlin
import org.bson.codecs.pojo.annotations.BsonId
import org.litote.kmongo.* //NEEDED! import KMongo extensions
import org.litote.kmongo.util.ObjectMappingConfiguration
import kotlin.test.Test
import kotlin.test.assertEquals
import kotlin.test.assertNotNull

class AppTest {

    data class Document(
            @BsonId val id: Int,
            val field1: String,
            val field2: String?
    )

    @Test fun testMongoBehaviours() {
        val client = KMongo.createClient()
        val database = client.getDatabase("test")
        val col = database.getCollection<Document>()
        ObjectMappingConfiguration.serializeNull = false // This is the default value anyway. Just being verbose
        run {
            col.save(Document(1, "Hello1", null))
            val document = col.findOne(Document::id eq 1)
            assertNotNull(document)
            assertEquals("Hello1", document.field1)
            assertEquals(null, document.field2)
        }
        run {
            col.save(Document(1, "Hello2", "World2"))
            val document = col.findOne(Document::id eq 1)
            assertNotNull(document)
            assertEquals("Hello2", document.field1)
            assertEquals("World2", document.field2)
        }
        run {
            col.save(Document(1, "Hello3", null))
            val document = col.findOne(Document::id eq 1)
            assertNotNull(document)
            assertEquals("Hello3", document.field1)
            assertEquals("World2", document.field2) // EXCEPTION THIS LINE. I expect this null value not to be serialized
        }
    }
}
```
I get
```log
java.lang.AssertionError: 
Expected :World2
Actual   :null
```

--- a/kmongo-native-mapping/src/main/kotlin/org/litote/kmongo/pojo/PojoClassMappingTypeService.kt
+++ b/kmongo-native-mapping/src/main/kotlin/org/litote/kmongo/pojo/PojoClassMappingTypeService.kt
@@ -52,7 +52,7 @@ internal class PojoClassMappingTypeService : ClassMappingTypeService {
     override fun filterIdToBson(obj: Any): BsonDocument {
         val bsonDocument = BsonDocument()
         val bsonWriter = BsonDocumentWriter(bsonDocument)
-        codecRegistryWithNullSerialization
+        ClassMappingType.codecRegistry(MongoClientSettings.getDefaultCodecRegistry())
             .get(obj.javaClass)
             ?.encode(
                 bsonWriter,

